crates/packager/src/config/mod.rs

```diff
@@ -288,6 +299,49 @@ impl DebianConfig {
     }
 }
 
+
+/// A list of dependencies specified as either a list of Strings
+/// or as a path to a file that lists the dependencies, one per line.
+#[derive(Debug, Clone, Deserialize, Serialize)]
+#[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
+#[serde(untagged)]
+#[non_exhaustive]
+pub enum Dependencies {
+    /// The list of dependencies provided directly as a vector of Strings.
+    List(Vec<String>),
+    /// A path to the file containing the list of dependences, formatted as one per line:
+    /// ```text
+    /// libc6
+    /// libxcursor1
+    /// libdbus-1-3
+    /// libasyncns0
+    /// ...
+    /// ```
+    Path(PathBuf),
```

Comment:
This seems a bit unconventional, how about adding support for specifying dependencies as environment variables, `CARGO_PACKAGER_DEB_DEPS="dep1,dep2,dep3"` and `CARGO_PACKAGER_PACMAN_DEPS="dep1,dep2,dep3"`?

Reply:
Sure, a pair of env vars would also work, especially for simple cases like the one I specified above. I can add that as well.

Would it be alright if we kept the file path option in there? There are a few reasons that come to mind, some from personal experience:
* Unix utils tend to interop more easily with files; env vars are harder to access & modify by comparison.
* Files are useful as intermediary artifacts, making it easy to manually inspect & modify them if needed in between various packaging steps
   * For example, if you have a 
* It can be tough to specify complex dependency conditions while using the right quotation & special char escaping syntax to marshal them properly into env vars
   * To clarify, if you have a complex dependency that uses special characters like `[`, `*`, `~`, `!`, etc, then it's easy to have a shell environment accidentally misinterpret those characters (especially for non-expert users), and especially when multiple different shell environments may be in use by various team members or build systems
       * Some examples of weird dependency strings are: `openjdk-7-jre-headless (>= 7~u3-2.1.1)`, `libc6 (>= 2.2.1), default-mta | mail-transport-agent`,`foo [linux-any], bar [any-i386], baz [!linux-any]`, `libssl1.0.0_1.0.1t-1+deb8u6_amd64.deb` etc.  These are all quite a bit easier to express in a plaintext file.
* Setting env vars differs across host OSes and can be (mildly) difficult to deal with in an abstract manner, and some runtime environments (chroots and the like) may clear or override env vars, which can be hard to debug.

Reply:
Thanks for explanation, let's go with your suggestion using a file